<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вычислительная гемодинамика</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/Resource.qrc">
    <normaloff>:/Application/icon 24 x 24.png</normaloff>:/Application/icon 24 x 24.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QVTKOpenGLNativeWidget" name="qvtkWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>964</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>Добавить в проект</string>
     </property>
     <property name="icon">
      <iconset resource="res/Resource.qrc">
       <normaloff>:/Application/layers.png</normaloff>:/Application/layers.png</iconset>
     </property>
     <addaction name="actionAddModel"/>
     <addaction name="actionAddMesh"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="menu"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="actionViewModel"/>
    <addaction name="actionViewMesh"/>
    <addaction name="actionViewResult"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="actionSaveAsImage"/>
    <addaction name="actionSaveAsTable"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="res/Resource.qrc">
     <normaloff>:/Application/info.png</normaloff>:/Application/info.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res/Resource.qrc">
     <normaloff>:/Application/open.png</normaloff>:/Application/open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res/Resource.qrc">
     <normaloff>:/Application/save.png</normaloff>:/Application/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res/Resource.qrc">
     <normaloff>:/Application/close.png</normaloff>:/Application/close.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res/Resource.qrc">
     <normaloff>:/Application/new.png</normaloff>:/Application/new.png</iconset>
   </property>
   <property name="text">
    <string>Создать...</string>
   </property>
  </action>
  <action name="actionSaveAsImage">
   <property name="icon">
    <iconset resource="res/Resource.qrc">
     <normaloff>:/Application/image.png</normaloff>:/Application/image.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как картинку...</string>
   </property>
  </action>
  <action name="actionSaveAsTable">
   <property name="icon">
    <iconset resource="res/Resource.qrc">
     <normaloff>:/Application/data table.png</normaloff>:/Application/data table.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как таблицу...</string>
   </property>
  </action>
  <action name="actionViewModel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Модель</string>
   </property>
  </action>
  <action name="actionViewMesh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Расчетная сетка</string>
   </property>
  </action>
  <action name="actionViewResult">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Результат</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="res/Resource.qrc">
     <normaloff>:/Application/reset.png</normaloff>:/Application/reset.png</iconset>
   </property>
   <property name="text">
    <string>Сбросить масштаб</string>
   </property>
  </action>
  <action name="actionAddModel">
   <property name="icon">
    <iconset resource="res/Resource.qrc">
     <normaloff>:/Application/model.png</normaloff>:/Application/model.png</iconset>
   </property>
   <property name="text">
    <string>Модель...</string>
   </property>
  </action>
  <action name="actionAddMesh">
   <property name="icon">
    <iconset resource="res/Resource.qrc">
     <normaloff>:/Application/mesh.png</normaloff>:/Application/mesh.png</iconset>
   </property>
   <property name="text">
    <string>Расчётную сетку...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QWidget</extends>
   <header>QVTKOpenGLNativeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
